store/streaming/README.md

```diff
@@ -0,0 +1,64 @@
+# State Streaming Service
+This package contains the constructors for the `StreamingService`s used to write state changes out from individual KVStores to a
+file or stream, as described in [ADR-038](../docs/architecture/adr-038-state-listening.md) and defined in [types/streaming.go](../types/streaming.go).
+The child directories contain the implementations for specific output destinations.
+
+Currently, a `StreamingService` implementation that writes state changes out to files is supported, in the future support for additional
+output destinations can be added.
+
+The `StreamingService` is configured from within an App using the `AppOptions` loaded from the app.toml file:
+
+```toml
+[store]
+    streamers = [ # if len(streamers) > 0 we are streaming
+        "file", # name of the streaming service, used by constructor
+    ]
+
+[streamers]
+    [streamers.file]
+        keys = ["list", "of", "store", "keys", "we", "want", "to", "expose", "for", "this", "streaming", "service"]
```

Comment:
What if I don't know all the keys or I don't want to provide them all? Can I use `*` or something to stream all keys?

Reply:
Yeah this needs to be fixed, the intention was that if the `keys` array is empty it would signify that we want to stream all keys (if we want to turn it off we would remove "file" from `store.streamers`) but the code is not doing that. And I think using '*' to signify we want to stream all keys is more explicit.

Reply:
Yeah can we go with that in this PR if it isn't too much work?

Reply:
Absolutely!